parts/configure-swarmmode-cluster.sh

```diff
@@ -131,7 +131,17 @@ echo "Installing and configuring Docker"
 installDocker()
 {
   for i in {1..10}; do
-    wget --tries 4 --retry-connrefused --waitretry=15 -qO- https://get.docker.com | sh
+    echo "Adding Docker’s public key for CS packages..."

```

Comment:
@amanohar this was required for installing Docker CS [1] (which is not supported anymore as only Docker CE and Docker EE remain). We should now change this for installing Docker CE or Docker EE.
[1] https://docs.docker.com/cs-engine/1.13/